diff -rN -u old-reviewed/src/Darcs/UI/PatchHeader.hs new-reviewed/src/Darcs/UI/PatchHeader.hs
--- old-reviewed/src/Darcs/UI/PatchHeader.hs	2018-03-03 18:19:38.418271172 -0500
+++ new-reviewed/src/Darcs/UI/PatchHeader.hs	2018-03-03 18:19:38.419271169 -0500
@@ -187,7 +187,7 @@
             | otherwise = fc
     writeTextFile f $ renderString
        $ text (reverse $ dropWhile isSpace $ reverse old)
-              -- ^^^ avoid empty line before instructions
+              -- avoid empty line before instructions
       $$ text "# Please enter the patch name in the first line, and"
       $$ text "# optionally, a long description in the following lines."
       $$ text "#"
